### PR TITLE
Develop 

### PR DESCRIPTION
**Pull request by nithyamn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# browserstack-percy-status-sync
+# BrowserStack - Percy Status Sync
+* Mark the test status on BrowserStack based on the Percy build state.
+
+## Pre-requisites
```
Comment (Rushabhshroff) [inline README.md:4]: lets add percy cli & related dependencies like Node JS?

File: src/test/java/MarkTestStatus.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        Request request = new Request.Builder()
+                .url("https://percy.io/api/v1/builds/"+buildID)
+                .get()
+                .addHeader("Authorization", "Token "+System.getenv("PERCY_TOKEN"))
+                .build();
+
+        Response response = client.newCall(request).execute();
+
+        JSONObject jsonObject = new JSONObject(response.body().string());
+        JSONObject jsonpObject2 = new JSONObject(jsonObject.get("data").toString());
```
Comment (Rushabhshroff) [inline src/test/java/MarkTestStatus.java]: If Build is not found this like will throw following exception
org.json.JSONException: JSONObject["data"] not found.
        at org.json.JSONObject.get(JSONObject.java:570)
        at MarkTestStatus.getPercyBuildStatus(MarkTestStatus.java:34)
        at MarkTestStatus.test(MarkTestStatus.java:14)
lets add null checks?

File: README.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+````
+export BROWSERSTACK_USERNAME=<YOUR_BROWSERSTACK_USERNAME> 
+export BROWSERSTACK_ACCESS_KEY=<YOUR_BROWSERSTACK_ACCESS_KEY>
+export PERCY_TOKEN=<PERCY_FULL_ACCESS_TOKEN>
+````
+## Steps to run
```
Comment (Rushabhshroff) [inline README.md:21]: npm install needs to be executed to install package.json dependencies

File: src/test/java/MarkTestStatus.java (hunk 2 above)
Comment (nithyamn) [reply src/test/java/MarkTestStatus.java]: added checks

File: README.md (hunk 3 above)
Comment (nithyamn) [reply README.md:21]: done

**Pull request merged**